app/controllers/apicasso/crud_controller.rb

```diff
@@ -179,18 +187,26 @@ def built_unpaginated
       { entries: @records, total: @records.size }
     end
 
-    # Parsed JSON to be used as response payload, with included relations
-    def include_relations
-      @records = JSON.parse(included_collection.to_json(include: parsed_include))
-    rescue ActiveRecord::AssociationNotFoundError, ActiveRecord::ConfigurationError
-      @records = JSON.parse(@records.to_json(include: parsed_include))
+    # Parse to include options
+    def include_options(object)
+      methods_include = []
+      associations_include = []
+      associations_array = resource.reflect_on_all_associations.map { |association| association.name.to_s }
+      parsed_include.map do |include_param|
+        if object.respond_to?(include_param)
+          if !associations_array.include?(include_param) ? methods_include << include_param : associations_include << include_param
```

Comment:
:197-198 - No need to open a `if` clause for a ternary operator, and you should avoid negation to keep a cleaner code.
Something in the lines of:
```
associations_array.include?(include_param) ? associations_include << include_param : methods_include << include_param
```